app/models/event.rb

```diff
@@ -116,6 +119,19 @@ def completed?
     status == "Completed"
   end
 
+  def use_template
```

Comment:
I changed it to `use_a_template`; This is where we're putting a template to use.